src/components/ReactSlider/ReactSlider.md

```diff
@@ -49,6 +49,7 @@ Double slider
     trackClassName="example-track"
     defaultValue={[0, 100]}
     ariaLabel={['Lower thumb', 'Upper thumb']}
+    ariaLabelledby={['Left thumb label', 'Right thumb label']}
```

Comment:
These examples do not have a separate visible label, so `ariaLabelledby` should not be used in any of these cases. If you want to create a separate example that shows how to use `ariaLabelledby`, that would be fine, but I would remove it from all the existing examples.

Reply:
No problem!

Reply:
Have made the changes as discussed hope they are suitable and fit in with the rest of the examples 😄